recipes/pgcgap/build.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh
 mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/lib/python3.6/site-packages/unicycler/
 
 cp pgcgap.pl $PREFIX/bin/pgcgap
+cp Functions/settings.py $PREFIX/lib/python3.*/site-packages/unicycler/
```

Comment:
What's the stuff with unicycler about? Isn't that a dependency?

Reply:
I have modified two parameters in the file "settings.py" of unicycler.  These two parameters were changed in unicycler version 0.4.8, but version 0.4.8 depends on python3.7, and PGCGAP is a python3.6 environment. So I want to replace the file in unicycler 0.4.7 with the file I edited.

I still have a question, when using bioconda, whether the dependent software is installed first or PGCGAP is installed first, it is related to whether the file I edited can successfully replace the file in the dependent software.

Or do you have better suggestions to solve this problem?

Thanks！

Reply:
Conda will install packages in a completely random order, so this won't work. It will also change the behavior of unicycler, which is something we can't permit (packages can't modify each other).

Reply:
I have already deleted the commands. Please have a check. Thanks!